exercises/all-your-base/example.h

```diff
@@ -0,0 +1,13 @@
+#ifndef ALL_YOUR_BASE_H
+#define ALL_YOUR_BASE_H
+
+#include <vector>
+#define EXERCISM_RUN_ALL_TESTS
+
+namespace all_your_base {
+
+    const std::vector<unsigned int> convert(unsigned int input_base, const std::vector<unsigned int>& input_digits, unsigned int output_base);
```

Comment:
A `const vector` return value is just odd, and I don't know of a legitimate reason for it. Please change to just a `vector`.

The formatting for the `const` input should be `std::vector<unsigned int> const& input_digits` to match the style